frontend/src/components/posts/PostUpdateForm.jsx

```diff
@@ -1,9 +1,13 @@
 // NPM Packages
 import React, { useState } from "react";
+import ResponsiveTextArea from '../ResponsiveTextArea';
 
 export default function PostUpdateForm({ onSubmit, post }) {
   // Local State
   const [contentText, setContentText] = useState(post.contentText);
+  const [length, setLength] = useState(post.contentText.length);
+  const postObject = require('../../assets/images/post.svg');
+  const postURL = postObject;
```

Comment:
this could be simplified to:
`const postURL = require('../../assets/images/post.svg');`

Reply:
Absolute yes :)